airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/scheduling/DefaultTaskQueueMapper.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.temporal.scheduling;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.airbyte.commons.temporal.TemporalJobType;
+import io.airbyte.config.Geography;
+import jakarta.inject.Singleton;
+import java.util.Map;
+
+@Singleton
+public class DefaultTaskQueueMapper implements TaskQueueMapper {
```

Comment:
I wonder if it would be easier to maintain a different approach here - my rough idea is that instead of maintaining a separate map for every job type going forward and adding a new RouterService method for each job type, perhaps the RouterService and TaskQueueMapper should be more opinionated and force a standard task queue format based on the geography and TemporalJobType name.

So, instead of having a separate map and method for every type of job, we could instead have a single method `getTaskQueue(TemporalJobType, Geography)` which returns the appropriate task queue. The DefaultTaskQueueMapper could simply return the TemporalJobType name without considering the Geography at all, and the CloudTaskQueueMapper could have a map from `Geography` to `taskQueueSuffix` that it adds to the TemporalJobType name to get the appropriate task queue.

Does that make sense? I haven't reviewed this full PR but I want to leave this comment up front to see what you think - I think it will make it easier to use/test/maintain the RouterService and TaskQueueMapper classes, and it forces some level of convention around how we name task queues for different jobs and geographies. Let me know what you think!

Reply:
Sure, thanks for the suggestion. That's done!